libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -1260,13 +1262,6 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
         _PITCHminf = MAX(_land_pitch_min, _PITCHminf);
     }
 
-    if (_landing.is_flaring()) {
```

Comment:
Its not clear to me at first glace why this has been removed?

Reply:
The flare specific setting of _PITCHmaxf  is already done at line 1237

Reply:
I should have split this out into a separate commit.

Reply:
basically, its just a consolidation using the above code. Same goes with the extra _landing.is_on_approach() call that could just get rolled into the same section above.

Reply:
I've split this change into a separate commit

Reply:
Ah, just outside the little window github shows us ;).